File: src/main/java/no/nav/syfo/rest/domain/RSEvaluering.java
package no.nav.syfo.rest.domain;


import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.Accessors;

@Data
@Accessors(fluent = true, chain = true)
@EqualsAndHashCode
public class RSEvaluering {

    public String effekt;
    public String hvorfor;
    public String videre;
    public boolean interneaktiviteter;
    public boolean ekstratid;
    public boolean bistand;
    public boolean ingen;
}

File: src/main/kotlin/no/nav/syfo/person/controller/RSEvaluering.kt
package no.nav.syfo.person.controller

data class RSEvaluering(
    val effekt: String? = null,
    val hvorfor: String? = null,
    val videre: String? = null,
    val interneaktiviteter: Boolean = false,
    val ekstratid: Boolean = false,
    val bistand: Boolean = false,
    val ingen: Boolean = false
)
